tests/django_app/manage.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!../../env/bin/python
```

Comment:
perhaps, but where would it live? It (and related flask/wsgi/celery apps) would need to be in another top level dir, not sure I see value in that.

FWIW, I explicitly exclude it from the test search path in pytest (see setup.cfg/norecursedirs)